dashboard/config/environments/adhoc.rb

```diff
@@ -36,12 +36,6 @@
   # the I18n.default_locale when a translation can not be found).
   config.i18n.fallbacks = true
 
-  # Whether or not to display pretty apps (formerly called blockly).
-  config.pretty_apps = true
-
-  # Whether or not to display pretty shared js assets
-  config.pretty_sharedjs = true
-
```

Comment:
open for discussion :) I discovered this duplicate config while pushing this branch to an adhoc instance, and figured I'd clean up all the road blocks I ran into within this PR, especially since rolling back this PR if something goes wrong won't necessarily affect the actual firebase migration. Happy to split any or all of these into a different PR if you prefer.